gke/README.md

```diff
@@ -6,7 +6,19 @@ Instructions of the initial setup of the GCP project and GKE clusters for deploy
 
 * [Create a Google Cloud Project](https://cloud.google.com/resource-manager/docs/creating-managing-projects). If working on an existing project, you should have owner/editor role to perform the tasks.
 
-* Make a copy of the `config.yaml.tpl` as `config.yaml`.
+* Verify that yq version 3 is installed
```

Comment:
Can you added a item like this:

* Install [yq](https://mikefarah.gitbook.io/yq/v/v3.x/). Make sure the version is 3.x
```bash
yq --version
```

Reply:
changed